src/app/components/navigation/navigation.component.html

```diff
@@ -1,5 +1,5 @@
 <nav class="navbar navbar-light bg-light fixed-top navbar-expand-lg" [ngClass]="{'shrink': isShrunk}" >
-  <a href="https://github.com/pamasse/zonemaster-gui" class="github-corner d-none d-lg-block d-xl-block" aria-label="View source on Github">
+  <a href="https://github.com/zonemaster/" class="github-corner d-none d-lg-block d-xl-block" aria-label="View source on Github">
```

Comment:
Isn't it more correct to link to the zonemaster/zonemaster-gui repo?

Reply:
Go to zonemaster/zonemaster/README.md

Reply:
Done